compiler-core/src/language_server/code_action.rs

```diff
@@ -75,6 +75,37 @@ impl CodeActionBuilder {
     }
 }
 
+/// A little wrapper around LineNumbers to make it easier to build text edits.
+///
+struct LineNumbersHelper<'a> {
```

Comment:
This isn't a great name as `Helper` doesn't mean anything. I think this class is for constructing `TextEdits`? If so can we have a name relating to that.